src/ya-comiste-backoffice/pages/products.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import * as React from 'react';
+import Link from 'next/link';
+
+export default function ProductsPage(): React.Node {
+  return (
+    <div>
+      <Link href="/products/create">
+        {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+        <a>Create a new product</a>
```

Comment:
Maybe create a re-usable link components, so you don't have to eslint-disable this everywhere?

Reply:
I am thinking about this one. My plan is to introduce something like `@adeira/sx-design` where I share things used by the other private project I have. It should be focused mainly on SX and I am currently trying to figure out what do to with these Next.js specific things. 🤔 I'd like to leave it for later.